cypress/support/commands.js

```diff
@@ -58,21 +58,22 @@ Cypress.Commands.add(
       })
     )
 
+    // This movement seems to be breaking after the latest react and cypress updates
     // Let Leaflet know the mouse has started to move. The diff between
     // mousedown and mousemove event needs to be large enough so that Leaflet
     // will really think the mouse is moving and not that it was a click where
     // the mouse moved just a tiny amount.
-    cy.log('mousemove', {
-      clientX: center.x,
-      clientY: center.y + 5
-    })
-    canvas.dispatchEvent(
-      new MouseEvent('mousemove', {
-        clientX: center.x,
-        clientY: center.y + 5,
-        bubbles: true
-      })
-    )
+    // cy.log('mousemove', {
+    //   clientX: center.x,
+    //   clientY: center.y + 5
+    // })
+    // canvas.dispatchEvent(
+    //   new MouseEvent('mousemove', {
+    //     clientX: center.x,
+    //     clientY: center.y + 5,
+    //     bubbles: true
+    //   })
+    // )
```

Comment:
Should these be removed, or will they be restored?

Reply:
I didn't really want to remove these, as it was copied from a stackoverflow answer I linked above the beginning of the command. So I left a comment explaining why I removed it, but left it there in case it would be helpful later